install/helm/templates/setup-job.yaml

```diff
@@ -54,6 +55,41 @@ spec:
       imagePullSecrets:
         - name: {{ .Values.deployment.image.imagePullSecret }}
       {{- end }}
+      {{- if or .Values.persistence.enabled $usingSqlite }}
+      initContainers:
+        - name: init-database
+          {{- if .Values.deployment.image.digest }}
+          image: {{ .Values.deployment.image.registry }}/{{ .Values.deployment.image.repository }}@{{ .Values.deployment.image.digest }}
+          {{- else }}
+          image: {{ .Values.deployment.image.registry }}/{{ .Values.deployment.image.repository }}:{{ default "latest" .Values.deployment.image.tag }}
+          {{- end }}
+          imagePullPolicy: {{ .Values.deployment.image.pullPolicy }}
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
+            {{- if .Values.deployment.securityContext.enableRunAsUser }}
+            runAsUser: {{ .Values.deployment.securityContext.runAsUser }}
+            runAsGroup: {{ .Values.deployment.securityContext.runAsUser }}
+            {{- end }}
+            capabilities:
+              drop:
+                - ALL
+          command: ['sh', '-c']
+          args:
+            - |
+              if [ ! -f /data/.initialized ]; then
+                echo "Copying database directory from image..."
+                mkdir -p /data
```

Comment:
Do we need this mkdir as we are mounting /data path. Or these are two data directories?